tools/scripts/run-many/run-many.ts

```diff
@@ -6,10 +6,16 @@ const target = process.argv[2];
 const jobIndex = Number(process.argv[3]);
 const jobCount = Number(process.argv[4]);
 let base = process.argv[5];
+
 // in case base is not a SHA1 commit hash add origin
 if (!/\b[0-9a-f]{5,40}\b/.test(base)) base = 'origin/' + base;
+if(base.includes('0000000000000000')){
+  base='origin/main'
```

Comment:
it could also be `master`

Reply:
Maybe we could add a unit test for this?

Reply:
not sure a unit test makes sense here also I´m not sure wether this solution works at all